scripts/ci-checks-v2.sh

```diff
@@ -47,20 +54,20 @@ if [[ "${GITHUB_EVENT_NAME:-}" == 'pull_request' ]]; then
 
 else
   # If we are doing this locally or on merge queue, then check for all the V2 crates
-  all_commands+=("cargo hack clippy --features 'v2,payment_v2,customer_v2' -p storage_impl")
-
-  common_command="cargo hack clippy --feature-powerset --depth 2 --ignore-unknown-features --at-least-one-of 'v2 ' --include-features '${v2_feature_set}'"
-  crates_to_include=""
+  all_commands+=("cargo hack clippy --features 'v2,payment_v2' -p storage_impl")
+  common_command="cargo hack clippy --feature-powerset --depth 2 --ignore-unknown-features --at-least-one-of 'v2 '"
   while IFS= read -r crate; do
     if [[ "${crate}" != "storage_impl" ]]; then
-      crates_to_include+="--package '${crate}' "
+      crates_to_include=""
```

Comment:
We don't need this line?

Reply:
We need this line
`crates_to_include=""`
This is to refresh the array, incase if we are not doing this, features will be appended and this will result in something in like:
```
1st loop iteration (1st command to be executed)
--features "a,b,c" --package a

2nd loop iteration (2nd command to be executed)
--features "a,b,c" --package a --features "a,b,d" --package b
```

So this ensures proper decoupling of the commands.